NEWS

```diff
@@ -1,33 +1,30 @@
-libmongoc 1.24.0 (Unreleased)
```

Comment:
The 1.24.0 release notes were intended to act as a reminder for the 1.24.0 release. I suggest keeping the 1.24.0 notes.